widgets/blueprintSources/src/BlueprintSources.tsx

```diff
@@ -76,8 +111,9 @@ export default function BlueprintSources({ data, toolbox, widget }) {
                     </NodesTree.Node>
                 );
             }
+            const mainYamlFilePath = `${data.blueprintTree.children[0].key}/${data.yamlFileName}`;
```

Comment:
I guess `data.blueprintTree.children[0].key` is the name of the root folder in the blueprint archive, right?
I think it'll make it a bit more clear if we have a const for that. WDYT about something like:
```suggestion
            const blueprintRootDirectory = data.blueprintTree.children[0].key;
            const mainYamlFilePath = `${blueprintRootDirectory}/${data.yamlFileName}`;
```
?